treeherder/perf/management/commands/synthesize_backfill_report.py

```diff
@@ -67,3 +68,16 @@ def _validate_args(self, frameworks: List[str], repositories: List[str]):
         if repositories:
             if not set(repositories).issubset(set(Command.repos_to_retrigger_on)):
                 raise ArgumentError('Unknown repository provided.')
+
+    def _queue_is_too_loaded(self, provisioner_id: str, worker_type: str, acceptable_limit=1000) -> bool:
+        '''
+        Helper method for PerfSheriffBot to check load on processing queue.
+        Usage example: _queue_is_too_loaded('gecko-3', 'b-linux')
+        :return: True/False
+        '''
+        tc = TaskclusterConfig('https://community-tc.services.mozilla.com')
+        tc.auth(client_id='PerfSheriffBot_client_id',
```

Comment:
We should be grabbing the credentials in a more standardized way. Check `treeherder/perf/investigation_aids.py` from [PR 6037](https://github.com/mozilla/treeherder/pull/6037).

Reply:
I took them from Django settings for the moment.
I saw that [TaskclusterModel](https://github.com/mozilla/treeherder/pull/6037/files?file-filters%5B%5D=.py#diff-d5a47a9cadc55d4d6530841371aa0dadR22) also keep these credentials.
After integration of both PRs we can use `TaskclusterModel` for `client_id` and `access_token`.